pytest_selenium/drivers/cloud.py

```diff
@@ -17,15 +17,15 @@ class Provider(object):
 
     @property
     def driver(self):
-        return type(self).__name__
+        return type(self).__name__.lower()
 
     @property
     def name(self):
         return self.driver
```

Comment:
Can we switch this around so this returns `type(self).__name__` and `driver` returns `self.name.lower()`?

Reply:
Agreed.